test/e2e/util.go

```diff
@@ -481,9 +482,14 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	}
 	kubectlArgs := append(defaultArgs, args...)
 
-	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
-	Logf("Running '%s %s'", cmd.Path, strings.Join(cmd.Args, " "))
+	//Allow users to specify KubectlPath at commandline... so that we can test kubectl (the binary)
+	//Or so that, in the dev mode, we can test the kubectl shell script.
+	cmdToRun := testContext.KubectlPath
```

Comment:
I think the best thing might be to revert to using the kubectl binary as we did before my change, rather than passing through cluster/kubectl.sh.

So basically, change the parameter default to "kubectl" and I think we're probably good. (Only issue is that it might break Jenkins, if kubectl isn't properly in the `PATH` anymore.)